src/test/Makefile

```diff
@@ -514,11 +514,11 @@ COMMON_FLAGS += -pthread
 endif
 
 # Flags passed to the C compiler.
-C_FLAGS = $(COMMON_FLAGS) \
+C_FLAGS = $(COMMON_FLAGS) $(EXTRA_FLAGS) \
```

Comment:
Where is `$(EXTRA_FLAGS)` defined?

Reply:
Missed from my local test. Should never be commited.